assets/js/components/link.js

```diff
@@ -73,6 +75,12 @@ class Link extends Component {
 				{ ...extraProps }
 			>
 				{ children }
+
+				{ external && (
+					<VisuallyHidden as="span" className="screen-reader-text">
```

Comment:
Using WordPress's `VisuallyHidden` is a `div` and not a `span` makes it less useful (as we'll frequently want visually-hidden text to accompany other text inside a block-level element that shouldn't hold a `div`, like here or in an `li`/`a` tag).

That's not the end-of-the-world, but combined with the custom `className` used here, there's really no reason to use `VisuallyHidden` over a `span`. It communicates intent, but it's not actually `VisuallyHidden` on its own at all.

From the looks of it, we have many other `span` elements using `screen-reader-text`, which isn't even defined in our CSS anyway. Importing `VisuallyHidden` here buys us nothing but inflates the size of our bundle, so let's replace `<VisuallyHidden as="span" className="screen-reader-text">` with a `<span className="screen-reader-text">` for now.

If there's demand for it we can make a follow-up issue to add our own `VisuallyHidden` component that's lighter-weight than Gutenberg's and will suit our CSS/usage. I've filed #2165 to track this.

Reply:
Makes sense @tofumatt . I've updated the PR. Can you check again?